src/azure-cli-core/azure/cli/core/extension/__init__.py

```diff
@@ -303,7 +303,15 @@ def get_extension_modname(ext_name=None, ext_dir=None):
 
 def get_extension_path(ext_name):
     # This will simply form the path for a WHEEL extension.
-    return os.path.join(EXTENSIONS_DIR, ext_name)
+    ext_sys_path = os.path.join(EXTENSIONS_SYS_DIR, ext_name)
+    ext_path = os.path.join(EXTENSIONS_DIR, ext_name)
+    return ext_path if os.path.isdir(ext_path) else (
```

Comment:
add `os.path.isdir(ext_path)` check here, is it because previous code may case potential bug ?

Reply:
Previous code use `get_extension_path` both before install when this path does not exist and when loading extensions. I splitted it to use the below `build_extension_path` for building the path before install, and to use this `get_extension_path` only for loading extensions. So we can add the isdir check here now.